packages/lib/src/vis/matrix/StickyGrid.tsx

```diff
@@ -27,26 +27,36 @@ const StickyGrid = forwardRef<HTMLDivElement, Props>((props, ref) => {
     <div
       ref={ref}
       className={styles.stickyGrid}
-      style={style}
+      style={{
+        ...style,
+        width: (style.width as number) + rowHeaderCellsWidth, // account for row header cells
+        height: (style.height as number) + cellSize.height, // account for column header cells
```

Comment:
To follow on my previous comment, I would be inclined to have a `columnHeaderCellsHeight` here for example